src/os/bluestore/BlueStore.h

```diff
@@ -1863,6 +1863,9 @@ class BlueStore : public ObjectStore,
 
   std::atomic<uint64_t> max_blob_size = {0};  ///< maximum blob size
 
+  std::atomic<uint64_t> total_ios = {0};
```

Comment:
i agree we should make this non-atomic.  also define it next to the kv_* stuff and comment that it is defined as the sum of ios and costs for kv_queue.  Perhaps these should be name kv_queue_ios and kv_queue_cost